specification/mixedreality/resource-manager/Microsoft.MixedReality/stable/2021-01-01/common.json

```diff
@@ -0,0 +1,167 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Mixed Reality",
+    "description": "Mixed Reality Resource Provider API",
+    "version": "2021-01-01"
+  },
+  "paths": {},
+  "definitions": {
+    "AccountKeys": {
+      "description": "Developer Keys of account",
+      "type": "object",
+      "properties": {
+        "primaryKey": {
+          "description": "value of primary key.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "secondaryKey": {
+          "description": "value of secondary key.",
+          "readOnly": true,
+          "type": "string"
+        }
```

Comment:
Are these a secret? Then please annotate with x-ms-secret. Also please make sure they are not exposed via GET. https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html#common-issues-found-in-review

Reply:
Thanks, yes, these are secrets, I've added x-ms-secret to the latest version. These are exposed through a POST operation,

Reply:
Hm, so this is only ever returned in a listkeys operation. How do I fix the problem modelValidation where it does not like the example output? I tried looking in the other specs and tried  "************" but this did not work.

Reply:
@j5lim any advice on this? should I remove x-ms-secret since it is only used in responses?

Reply:
I see this issue in other PR tool. It may be a tool issue.

Reply:
@roytan-microsoft , this is by design. Please suppress this error in `readme.md` in case of `post` scenario like below.
## Suppression
``` yaml
directive:
  - suppress: SECRET_PROPERTY
    from: xxx.json
    where:
      - $.definitions.xxx
```

Reply:
Thanks @raych1 I have added the supression